docs/send-data/opentelemetry-collector/data-source-configurations/collect-logs.md

```diff
@@ -5,89 +5,212 @@ sidebar_label: Collect Logs
+* __receivers__: 
+  * `filelog/custom_files:` Unique name defined for the filelog receiver. You can add multiple file receivers as filelog/2 , filelog/prod etc.
+  * `include:` Lets the Filelog Receiver know where the log file is placed. Make sure that the collector has permissions to access the files; otherwise, it will not be collected.
+  * `include_file_path_resolved: true` Adds a `log.file.path_resolved` attribute to the logs that contain the whole path of the file.
+  * `storage: file_storage:` Prevents the receiver from reading the same data over and over again on each otelcol restart. Extension defined by default in sumologi.yaml
+ 
+* __processors__: 
+  * `resource/1` Name of the processor that uses `attributes` to add \_sourceCategory field with value `application_logs_prod`
+ 
+* __exporters__: 
+  *  `sumologic` Sends data to registered Sumologic Organization.  Auto configured in sumologic.yaml during installation using script.
+ 
+* __service__: 
+  *  `logs/pipeline1:`  Pipeline glues together what data to collect(receiver), how to process the data(processors) and where to send logs(exporter).
 
-The remaining processors in pipeline are from `sumologic.yaml` file and should be applied for better performance of collector and use of Sumo Logic platform.
+
+
+The remaining processors in pipeline are from `sumologic.yaml` file and should be applied for better performance of the collector.
 
 :::note
-The receiver (`filelog/custom_files`) and pipeline (`logs/custom_files:`) names should be unique across all configuration files to avoid conflicts and unexpected behavior.
+The receiver (`filelog/custom_files`) ,  (`resource/processor1`) and  pipeline (`logs/pipeline1:`) names should be unique across all configuration files to avoid conflicts and unexpected behavior.
 :::
 
-### Keeping track of position in files
 
-By default, the Filelog receiver watches files starting at their end (the default is `start_at: end`), so nothing will be read after the otelcol process starts until new data is added to the files. To change this, add `start_at: beginning` to the receiver's configuration.
+For more details, see the [Filelog Receiver documentation][filelogreceiver_readme].
 
-To prevent the receiver from reading the same data over and over again on each otelcol restart, also add the [File Storage extension][filestorageextension_docs] which is already defined in `sumologic.yaml`, that will allow Filelog receiver to persist the current position in watched files between otelcol restarts. Here's an example of such a configuration:
+## Windows Log Event Receiver
 
+Windows Log Event Receiver reads and parses logs from windows event log API to collect local events as you see on Windows Event Viewer.
+
+Following configuration demonstrates:
+ 1. Collect : Collect Application, Security and System channels 
+ 2. Transform : Set \_sourceCatetgory field to "windows_event_log_prod" 
+ 3. Export : Send data to authenticated Sumologic organization
```

Comment:
See https://github.com/SumoLogic/sumologic-documentation/pull/2501/files#r1208088856 for formatting